examples/raw-window-handle-with-wgpu/main.rs

```diff
@@ -102,15 +102,18 @@ fn main() -> Result<(), String> {
             alpha_to_coverage_enabled: false,
         },
         multiview: None,
+        cache: None,
     });
 
     let mut config = wgpu::SurfaceConfiguration {
         usage: wgpu::TextureUsages::RENDER_ATTACHMENT,
-        format: surface.get_supported_formats(&adapter)[0],
+        format: wgpu::TextureFormat::Bgra8UnormSrgb,
```

Comment:
I guess this maybe shouldn't be hardcoded?

Reply:
Well it's just an example, and I didn't see any newer version of the get_supported_formats function. Tbh I didn't do much looking, but all the official examples in wgpu I saw just hardcoded it

Reply:
Okay actually I have a working solution for this